service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseController.java

```diff
@@ -33,96 +31,41 @@
 
     private static final String MIGRATION_ID_KEY = "migrationId";
 
-    private static final UUID BUNDLE_ID = UUID.fromString("b02898e7-46dc-47ce-9639-9e5b04d03b9e");
-    private static final UUID C2_ID = UUID.fromString("4b725c8a-3496-4f28-83f1-95d4838a533a");
-    private static final String C2_DOC_ID = "b444c4fb-362b-4e27-b7d8-61996b3f6e0d";
-    private static final String FAMILY_MAN_CASE_NUMBER = "SA20C50050";
-    private static final int BUNDLE_SIZE = 3;
-    private static final int BUNDLE_INDEX = 2;
-
-    private static final String FAMILY_MAN_ID = "DE21C50016";
-    private static final UUID DOCUMENT_TO_REMOVE_UUID = UUID.fromString("2acc1f5f-ff76-4c3e-b3fc-087ebebd2911");
-
     @PostMapping("/about-to-submit")
     public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody CallbackRequest callbackRequest) {
         CaseDetails caseDetails = callbackRequest.getCaseDetails();
         Object migrationId = caseDetails.getData().get(MIGRATION_ID_KEY);
 
-        if ("FPLA-3135".equals(migrationId)) {
-            run3135(caseDetails);
-        }
-
-        if ("FPLA-3125".equals(migrationId)) {
-            run3125(caseDetails);
+        if ("FPLA-3092".equals(migrationId)) {
+            run3092(caseDetails);
         }
 
         caseDetails.getData().remove(MIGRATION_ID_KEY);
         return respond(caseDetails);
     }
 
-    private void run3125(CaseDetails caseDetails) {
-        CaseData caseData = getCaseData(caseDetails);
-
-        if (!Objects.equals(FAMILY_MAN_CASE_NUMBER, caseData.getFamilyManCaseNumber())) {
-            throwException("family man case number", FAMILY_MAN_CASE_NUMBER, caseData.getFamilyManCaseNumber());
-        }
-
-        List<Element<AdditionalApplicationsBundle>> bundles = caseData.getAdditionalApplicationsBundle();
-
-        if (BUNDLE_SIZE != bundles.size()) {
-            throwException("additional applications bundle size", BUNDLE_SIZE, bundles.size());
-        }
-
-        Element<AdditionalApplicationsBundle> bundle = bundles.get(BUNDLE_INDEX);
-
-        validateBundle(bundle);
-
-        bundles.remove(BUNDLE_INDEX);
+    private void run3092(CaseDetails caseDetails) {
+        final String familyManId = "CF20C50063";
```

Comment:
please double check that family man number is correct as story mention 2 different